Options.py

```diff
@@ -1449,6 +1461,8 @@ class PerGameCommonOptions(CommonOptions):
     start_location_hints: StartLocationHints
     exclude_locations: ExcludeLocations
     priority_locations: PriorityLocations
+    local_locations: LocalLocations
```

Comment:
I'm not sure about this being forced on every game especially without a public discussion about this.

Reply:
(For summary purposes, it was discussed in the Discord. People seemed generally favorable and no one objected.)